README.md

```diff
@@ -248,3 +248,47 @@ To compare all mails on the list against upstream:
 [1]: https://public-inbox.org/README.html
 [2]: https://github.com/xai/nntp2mbox
 [3]: https://git.kernel.org/pub/scm/public-inbox/
+
+PaStA and Patchwork
+--------------------
+The results of PaStA's analyses can be used by [Patchwork](http://jk.ozlabs.org/projects/patchwork/).
+
+## Setting up PaStA and Patchwork
+Assuming a working setup of PaStA already exists, here are the steps necessary for Patchwork integration
+
+1. Install Patchwork on your system, following the guidelines in Patchwork's [documentation](https://patchwork.readthedocs.io/en/latest/development/installation/)
+
+2. Start a shell inside Patchwork's docker container with `docker-compose run --rm web --shell`
+
+3. Bring up a Patchwork development server, by running
+`./manage.py runserver 0.0.0.0:8000` inside the shell started in step 2. You should now have a Patchwork instance
+running at `<Patchwork-Docker-Container-IP-address>:8000` on your host. The Patchwork container's IP address can be
+found using `ifconfig` command on Linux distributions.
+
+4. Start PaStA's docker container on the same network as the Patchwork one by running the command:
+`docker run -it --rm --network patchwork_default --name pasta -v </path/to/PaStA>:/home/pasta pasta:latest`
+
+5. Set the Patchwork specific settings in config:
+```
+[mbox]
+...
+
+USE_PATCHWORK_ID = true
```

Comment:
no lomger required

Reply:
> We should allow to have no initial_archive set in the configuration file.

So do you mean instead of checking for empty string in `initial_archive` property, we let the user completely remove that key?

It seems like the another way to do the same thing. Are there any advantages to remove the key instead of setting it to an empty string?

Reply:
Usually people comment out configuration options that they don't need. This is what I intuitively tried. I commented out initial_archive after the first run. Resulted in a crash of PaStA.

Reply:
initial_archive key is associated with a projectid and list email. ie it is part of the same object.
So I dont think that can be commented out.

If we keep a separate array of initial_archives and separate array of projects that might be troublesome as both the arrays may have conflicting information